phd-tests/tests/testdata/dirt.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env sh
+# a simple script for testing migration of running process memory.
+#
+# this script will write a bunch of "hello"s to an in-memory string, then
+# suspend to wait for a migration to occur. after the migration happens, the
+# script can be foregrounded, and it will then check that the data is still
+# there.
+dirt="hello"
+len=8192 # 8 KiB ought to be enough for anyone
+
+# store 8k of data in memory
+i=0
+while [ "$i" -lt "$len" ]
+do
+    dirt="$dirt
+hello"
+    i=$(( i + 1 ))
+done
+
+echo "made dirt"
+
+# suspend this process and wait for it to be resumed before checking that the
+# data still exists.
+#
+# N.B.: posix sh doesn't have a suspend builtin, but we can make our own!
```

Comment:
No action required here, but we've really got to start running CI on something other than Alpine.

Maybe I'll make another project board...